wwwroot/inc/snmp.php

```diff
@@ -2495,6 +2501,12 @@
 		'text' => 'Cisco 878 ISR: 4 RJ-45/10-100TX',
 		'processors' => array ('catalyst-chassis-any-100TX'),
 	),
+	'9.1.574' => array
+	(
+    	'dict_key' => 143,
+    	'text' => 'Cisco Catalyst 3750 Metro Series Switches, 24 ports 10/100, 2 port SFP Gigabit Ethernet, 2 port SFP',
+    	'processors' => array ('cisco-25-to-28-1000SFP','catalyst-stack-any-100TX'),
```

Comment:
for the text, please follow this formatting pattern:
```
WS-C2960S-F48LPS-L: 48 RJ-45/10-100TX + 4 SFP/1000
ME-C3750-24TE-M: 24 RJ-45/10-100TX + 4 SFP/1000
```


snippet from the ticket:
```
IF-MIB::ifDescr.10023 = STRING: FastEthernet1/0/23
IF-MIB::ifDescr.10024 = STRING: FastEthernet1/0/24
IF-MIB::ifDescr.10101 = STRING: GigabitEthernet1/0/1
IF-MIB::ifDescr.10102 = STRING: GigabitEthernet1/0/2
IF-MIB::ifDescr.10301 = STRING: GigabitEthernet1/1/1
IF-MIB::ifDescr.10302 = STRING: GigabitEthernet1/1/2
```
`catalyst-stack-any-100TX` is correct because it matches `FastEthernet(\d+)/(\d+)/(\d+)`
`cisco-25-to-28-1000SFP` won't work because it matches `GigabitEthernet1/1/(25|26|27|28)` while the SNMP output shows `GigabitEthernet1/x/x`.
for the SFP ports you can use `catalyst-stack-1-to-4-SFP`